scapy/layers/tls/record.py

```diff
@@ -119,7 +119,8 @@ def getfield(self, pkt, s):
             remain, ret = s[:l], s[l:]
 
         if remain == b"":
-            if pkt.tls_session.tls_version > 0x0200 and pkt.type == 23:
+            if ((pkt.tls_session.tls_version or 0x0303 > 0x0200) and
```

Comment:
Operator precedence is very confusing here. Care to add parentheses? ` ((version or 0x0303) >= 0x0200)`
Same below.